src/index.js

```diff
@@ -4,9 +4,9 @@ import * as ReactDOM from "react-dom";
 import * as Scrivito from "scrivito";
 import "./Objs";
 import "./Widgets";
+import "./assets/stylesheets/index.scss";
 import App from "./App";
 import "./config";
-import "./assets/stylesheets/index.scss";
```

Comment:
Why is this change needed? Which styling in `index.scss` overwrites styles of `Navigation.scss`?

My goal would be to have `index.scss` loaded _last_. We either have to fix the stylings in `Novigation.scss` accordingly or import `Navigation.scss` via `index.scss` and not via the js component.

Reply:
`.menu-toggle .fa` & `.navbar-brand` at least 

<img width="818" alt="Screenshot 2019-09-04 at 13 46 41" src="https://user-images.githubusercontent.com/44314538/64252359-f7807100-cf1a-11e9-98f2-1cbf13621e4f.png">
<img width="540" alt="Screenshot 2019-09-04 at 13 47 01" src="https://user-images.githubusercontent.com/44314538/64252360-f7807100-cf1a-11e9-80b8-da6801dd3396.png">

so maybe I can add it to imports in index.scss as you suggested?

Reply:
If you don't find another way simply import it via `index.scss`.